packages/js/examples/react-audio/stories/components/Utilities.js

```diff
@@ -108,6 +109,45 @@ function Utilities({ environment, username, password }) {
     });
   };
 
+  const newCall = async () => {
```

Comment:
Does this need to be in `Utilities` if there's already a dialer component? Seems like a lot of code duplication in Storybook.

Reply:
I thought that `Utilities` was to do a playground, for example on the `newCall` I'm overwriting  the `onNotification` to test, and I tested to select a specific `webcam` when I make a call specifying the `camId`

```
const options = {
      destinationNumber: '+18564440362',
      callerName: 'Telnyx Caller',
      onNotification,
     camId: 'd346d0f78627e3b808cdf0c2bc0b25b4539848ecf852ff03df5ac7545f4f5398'
    };
```

I can remove with you want.